lib/skill/actions.ts

```
@@ -0,0 +1,175 @@
+ */
+export const fetchGlobalSkills = async (): Promise<GlobalSkill[]> => {
+	try {
+		const data = await api.get<GlobalSkill[]>('/skill/')
+
+		// Map the data and log each skill parsing result
+		const parsedSkills = data.map(skill => {
+			try {
+				const parsed = GlobalSkillSchema.parse(skill)
+				return parsed
+			} catch (parseError) {
+				throw parseError
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)